pkg/apis/flyteworkflow/v1alpha1/workflow.go

```diff
@@ -34,6 +34,8 @@ type FlyteWorkflow struct {
 	// Value must be a positive integer.
 	// +optional
 	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,omitempty"`
+	// Interruptible value set here is default value used for nodes unless explicitly set at the node level.
+	Interruptible bool `json:"interruptible,omitempty"`
```

Comment:
This is default behavior for the nodes (that don't define interruptible for themselves) in this WF. What do we name it- NodesInterruptible?
cc @migueltol22 @EngHabu @kumare3

Reply:
Maybe add a `NodeDefaults` struct and add `Interruptible` in there?